### DISCUSSION TITLE
Update regarding your ticket with HireVue Support! Re: Re: Congrats - you have a digital assessment #436440

### DISCUSSION THREAD

**Discussion by ahmeraliraza2** [community/community#42914] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![Uploading Screenshot_20221228_154222.jpg…]()

**Comment by jfullstackdev** [comment, marked as answer]
the image is broken, we can not be of help without any further details

**Reply by ahmeraliraza2** [reply]
Hi Sir
How the problem will solve???

**Reply by jfullstackdev** [reply]
paste another image that is not corrupted, so we can see it too

**Comment by ahmeraliraza2** [comment]
![Screenshot_20221228_155118](https://user-images.githubusercontent.com/121662943/210065415-7ca84495-9db8-4f69-9353-b4d4e0c14d9a.jpg)

**Comment by ahmeraliraza2** [comment]
![Uploading Screenshot_20221230_130346.jpg…]()
